release/_bom.sh

```diff
@@ -224,58 +224,55 @@ function generate_pom_release_dxp_bom_third_party {
 		-e "w ${pom_file_name}" \
 		"${_RELEASE_TOOL_DIR}/templates/release.dxp.bom.third.party.pom.tpl" > /dev/null
 
-	local property_files=(
-		"${_PROJECTS_DIR}/liferay-portal-ee/lib/development/dependencies.properties"
-		"${_PROJECTS_DIR}/liferay-portal-ee/lib/portal/dependencies.properties"
-	)
+	IFS=$'\n' read -d '' -r -a development_properties < "${_PROJECTS_DIR}/liferay-portal-ee/lib/development/dependencies.properties"
```

Comment:
@lucasmiranda0, do we need the `$`? Can we keep just the `IFS='\n'`?

In addition, please rename to `dependencies_properties` instead of `development_properties` to keep consistency with the file we're collecting the info. :wink: 

One more thing, can you use `-ra` instead of `-r -a`? You did this the `for` below, so let's try to keep the consistency.